controller.go

```diff
@@ -480,3 +492,72 @@ func (c *Controller) removeMember(memberDID string) (map[string]string, error) {
 	}
 	return map[string]string{"status": "ok"}, nil
 }
+
+func (c *Controller) startBitcoind() (map[string]interface{}, error) {
+	req, err := http.NewRequest("POST", viper.GetString("bitcoind_ctl.endpoint")+"/start", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	var resBody json.RawMessage
+	statusCode, err := c.doHttpRequest(req, &resBody)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return map[string]interface{}{
+		"statusCode":   statusCode,
+		"responseBody": resBody,
+	}, nil
+}
+
+func (c *Controller) stopBitcoind() (map[string]interface{}, error) {
+	req, err := http.NewRequest("POST", viper.GetString("bitcoind_ctl.endpoint")+"/stop", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	var resBody json.RawMessage
+	statusCode, err := c.doHttpRequest(req, &resBody)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return map[string]interface{}{
+		"statusCode":   statusCode,
+		"responseBody": resBody,
+	}, nil
+}
+
+func (c *Controller) getBitcoindStatus() (map[string]interface{}, error) {
+	req, err := http.NewRequest("GET", viper.GetString("bitcoind_ctl.endpoint")+"/status", nil)
+	if err != nil {
+		log.WithError(err).Error("fail to create bitcoind-ctl api request")
+		return nil, fmt.Errorf("fail to create bitcoind-ctl api request")
+	}
+	var resBody json.RawMessage
+	statusCode, err := c.doHttpRequest(req, &resBody)
+	if err != nil {
+		log.WithError(err).Error("fail to call bitcoind-ctl api")
+		return nil, fmt.Errorf("fail to call bitcoind-ctl api")
+	}
+	return map[string]interface{}{
+		"statusCode":   statusCode,
+		"responseBody": resBody,
+	}, nil
+}
+
+func (c *Controller) doHttpRequest(req *http.Request, resBody *json.RawMessage) (int, error) {
```

Comment:
I think it would be better to return the body in return value instead of implicit write to the variable pointer.